Homestead/readme.md

```diff
@@ -45,11 +45,10 @@ authorize: ~/.ssh/id_rsa.pub
 keys:
     - ~/.ssh/id_rsa
 
-# Install MongoDB, MariaDB, and Chrome WebDriver.
+# Install MongoDB & MariaDB.
 features:
     - mongodb: true
     - mariadb: true
-    - webdriver: true
```

Comment:
I removed this because we no longer run WebDriver tests from within the box (Laravel Dusk).